lib/Smokeping.pm

```diff
@@ -3741,6 +3911,14 @@ sub get_config ($$){
     if (not $cfg->{Presentation}{multihost} or not $cfg->{Presentation}{multihost}{colors}){
        $cfg->{Presentation}{multihost}{colors} = "004586 ff420e ffde20 579d1c 7e0021 83caff 314004 aecf00 4b1f6f ff950e c5000b 0084d1";
     }
+    # based on the current configuration - if influxdb support is enabled in the InfluxDB section, load the necessary influxdb perl modules
+    if(defined $cfg->{InfluxDB}{host}){
+	    do_debuglog("Found InfluxDB section. Loading InfluxDB modules");
+	    use InfluxDB::HTTP;
```

Comment:
these `use` statements get read at compile time, so you can as well move them to the top of the file.

Reply:
Thanks for the suggestion. In a previous attempt I wanted to load a module with a precision=s/ms/us value, but it didn't work. I'll move them on top.

Reply:
Ok, made the change here: https://github.com/oetiker/SmokePing/pull/215/commits/c4076fcdf9d3ca722df5dbe121bd1a77fac621c7